python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -426,6 +430,7 @@ def __new__(
         asset_events: Optional[
             Sequence[Union[AssetObservation, AssetMaterialization, AssetCheckEvaluation]]
         ] = None,
+        **kwargs,
```

Comment:
This is me just doing the [hidden_param](https://github.com/dagster-io/internal/discussions/10315) pattern by hand.